ui/components/ui/contract-token-values/contract-token-values.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CopyContractDetails from '../icon/copy-contract-details';
+import IconBlockExplorer from '../icon/icon-block-explorer';
+import Box from '../box/box';
+import Tooltip from '../tooltip/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Identicon from '../identicon/identicon.component';
+import Typography from '../typography/typography';
+import {
+  FONT_WEIGHT,
+  TYPOGRAPHY,
+  DISPLAY,
+  COLORS,
+  ALIGN_ITEMS,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
+import Button from '../button';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+
+export default function ContractTokenValues({ address, tokenName }) {
+  const t = useI18nContext();
+  const [copied, handleCopy] = useCopyToClipboard();
+
+  return (
+    <Box
+      display={DISPLAY.FLEX}
+      alignItems={ALIGN_ITEMS.CENTER}
+      justifyContent={JUSTIFY_CONTENT.CENTER}
+      className="contract-token-values"
+    >
+      <Box marginRight={2} marginTop={2} marginBottom={2}>
+        <Identicon address={address} diameter={24} />
+      </Box>
+      <Box>
+        <Typography
+          variant={TYPOGRAPHY.H2}
+          fontWeight={FONT_WEIGHT.BOLD}
+          color={COLORS.TEXT_ALTERNATIVE}
+          marginTop={0}
+          marginBottom={0}
+        >
+          {tokenName}
+        </Typography>
+      </Box>
```

Comment:
Do we need this extra wrapper?
```suggestion
      <Typography
        variant={TYPOGRAPHY.H2}
        fontWeight={FONT_WEIGHT.BOLD}
        color={COLORS.TEXT_ALTERNATIVE}
        marginTop={0}
        marginBottom={0}
      >
        {tokenName}
      </Typography>
```

Reply:
@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/48c24613122a69ea20b0ad6504a3586585b6be25).